clis/ph-cli/src/services/connect.ts

```diff
@@ -1,8 +1,29 @@
 import type { ConnectStudioOptions } from "@powerhousedao/builder-tools";
 import { startConnectStudio } from "@powerhousedao/builder-tools";
 
-export type ConnectOptions = ConnectStudioOptions["devServerOptions"];
+export type ConnectOptions = ConnectStudioOptions["devServerOptions"] & {
```

Comment:
Let's add these options to the `ConnectStudioOptions` type. So that the user can also set them on `vite.config.ts`